scripts/seed.ts

```
@@ -1,7 +1,8 @@
 import 'dotenv/config';
-import * as fs from 'fs';
 import { getDb } from '../lib/db';
 import { PoolStats } from '../lib/entities/PoolStats';
+import { CKPoolAPI } from '../lib/ckpool';
```

ESLint

[error] `../lib/ckpool` import should occur before import of `../lib/db`

(import/order)